tests/e2e/manifest/pod-curl-target-gw.yaml

```
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: curl-target-gw
+  namespace: target
+  labels:
+    app: curl-target-gw
+spec:
+  containers:
+  - name: curl
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)